### ISSUE TITLE
The map is blank

### ISSUE THREAD

**Issue by stsiushkevich** [JustFly1984/react-google-maps-api#185]
# Please provide an explanation

## Your Environment

os: windows

react version
"react": "16.6.0"

@babel version

@react-google-maps/api version
"@react-google-maps/api": "1.4.2"

## How is it behave?

I did all the instructions but my map is blank! I migrated from "react-google-maps": "9.4.5".

My code is:

```javascript
import React, { Component } from "react"

import cn from 'classname'

import {
    Marker,
    GoogleMap,
    LoadScript
} from '@react-google-maps/api'

import './Map.scss'

import config from '../../config'

export default class Map extends Component {
    map = null

    onLoad = map => {
        this.map = map
    }

    render () {
        const {
            className
        } = this.props

        return (
            <div style={{width: '100%', height: '100%'}} className={cn('Map', className)}>
                <LoadScript
                    id='script-loader'
                    loadingElement={<Loader/>}
                    googleMapsApiKey={config.google.maps.apiKey}>
                    <GoogleMap id='example-map' mapContainerStyle={{width: '100%', height: '100%'}}/>
                </LoadScript>
            </div>
        )
    }
}
```

![image](https://user-images.githubusercontent.com/12151060/61374616-7a833480-a8a5-11e9-84dc-b1e5feefb900.png)

**Comment by JustFly1984** [comment]
@stsiushkevich can you please make a basic example in codesandbox?

**Comment by scottydev** [comment]
Same happening to me. Using NEXT.js v9 and tried both the `useLoadScript` and `LoadScript` methods.

**Comment by JustFly1984** [comment]
@scottydev can you provide a basic example in codesandbox? We have published 1.5.0 version, please update and check

**Comment by scottydev** [comment]
Ok Stack Overflow helped me out: https://stackoverflow.com/a/11749810/1259863

Basically the `zoom` and `center` props are REQUIRED by Google Maps for initial load.
